graphql.go

```diff
@@ -327,6 +327,7 @@ type Errors []Error
 
 type Error struct {
 	Message    string                 `json:"message"`
+	Err        error                  `json:"-"`
```

Comment:
The error overlaps with the message. We should not expose it. Let's change it to a private property

Reply:
Thanks for the review.

I've made Err private.